src/js/hooks/list-pages/useCommonFilters.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react';
+
+import { useDispatch } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+
+const useCommonFilters = () => {
```

Comment:
What is the purpose of this hook? I am unsure if this hook reduces the amount of code in other components because you have to destructure the returned object, so it's also a lot of code. Basically, this hook doesn't contain any logic, just other hooks.

Reply:
I realized that on all of the list pages, the code that is inside this hook is redundant. Starting from now, we could reuse this hook for the future list pages (and refactor the existing ones) instead of copying and pasting those `useStates` etc. that are there.